### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by azat-co**
CC @AnnJohn @vicfriedman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
+
+## Instructions
+
+1. Enter Node REPL
+1. Print global object
+1. Narrow down to specific info of process (`global.process`)
```
Comment (victhevenot) [inline README.md]: what specific info? what does that info mean? what can we do with it?

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+1. Create a global variable `user` explicitly with `global.user = {admin: false}`
+2. Create a global variable implicitly with `var account = {balance: 1000}` 
+3. Open a new REPL and check for values of user and account (ReferenceError)
```
Comment (victhevenot) [inline README.md]: I would explain `ReferenceError` more in depth. prompt them to think about why they would get the error and what that means

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+
+## Objectives
+
+1. Use REPL
+1. Access global
+1. Understand scoping and explicit vs. implicit declaration
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
```
Comment (victhevenot) [inline README.md:13]: Make intro more informative - you are going to be using the REPL and node global scope to create bank user and bank account objects.

**Comment by azat-co** [comment]
updated according to the feedback CC @AnnJohn @vicfriedman

**Comment by victhevenot** [comment]
we've never talked about the `delete` method before to understand that it only operates on the global object. the comparison of `user` created as an attribute of `global` and `account` just a variable is lost because we've never been instrduced to `delete`

**Comment by azat-co** [comment]
updated CC @AnnJohn @vicfriedman

**Comment by victhevenot** [comment]
@azat-co, almost done here!

The intro needs some work - what do you mean `Understanding global and scoping will give you a good foundation for future development.` - We need to beef this up. Reiterate here why the global object (and the scope of the global object) is important to understand. Maybe give an example of a situation in which using `global` is really useful.

and on line 20, I would explain ReferenceError more in depth. tell them what that error means -that it's like  the Ruby, `undefined local method or variable` error. prompt them to think about why they would get the error and what that means  and then explain- the repl is  sandbox and doesn't persist variables across different instances.

**Pull request merged**